### DISCUSSION TITLE
'rgl_node_points_udp_publish' has been implemented?

### DISCUSSION THREAD

**Discussion by KASUYASU** [RobotecAI/RobotecGPULidar#234] [Q&A] [answered]
Hi,

Thank you for the wonderful library.

But I found the function "rgl_node_points_udp_publish" in the plugin script

https://github.com/tier4/AWSIM/blob/main/Assets/RGLUnityPlugin/Scripts/LowLevelWrappers/RGLNativeAPI.cs

Has this function been implemented? Not yet?

Kazu

**Comment by PawelLiberadzki** [comment, marked as answer]
Hi @KASUYASU,
I am a member of RGL team. Thank you for asking this question! UDP publishing was made exclusively as a part of our collaboration with TierIV. At this moment there is no plan to make it public. The visibility of this function's signature is only a matter of how RGL library is integrated with AWSIM.
Best regards,
Pawel

**Reply by KASUYASU** [reply]
Hi,

Thank you for your reply.
I think that if the UDP interface can be used, the cooperation with Tier IV's autoware will be more seamless between simulation and actual operation, but I am disappointed that it is currently not public.

Kazu

**Reply by PawelLiberadzki** [reply]
Thank you for the suggestion - we will probably change the interface here to make it more clear, but we will also follow your point during our internal discussion.

Best regards,
Pawel
